src/assets/style/header.scss

```diff
@@ -2,7 +2,7 @@
 
 .header__container {
   position: fixed;
-  z-index: 1;
+  z-index: 10;
```

Comment:
I have a div that overlaps the mascot after it is selected, and its z-index is also 1. When the header's z-index is 1, the header will be overwritten by the other div itself. Therefore, the z-index of the header must be greater than the z-index of the pick div. I convention the z-index ranks to be 1, 10, 100 in ascending order

Reply:
You don't need to set z-index too high just ```z-index: 1``` because you set ```position: fixed```. If you set value too high there will be a mess when your page has many layouts and hard to manage layout.